"""
We have many exception handling in python:
- Catching general Exceptions in Python
- Catching Specific Exception(value error, TypeError, ZeroDivisionError ... some built-in exception catching),
- Raising Exceptions in Python(the program still running, but the log will be showed in terminal)
- try with else clause(if the error not appear, then statement in else will be called)
- try...finally: This clause is executed no matter what, and is generally used to release external resources.

Reference : https://www.programiz.com/python-programming/exception-handling
"""


